src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,547 @@
+    {
+      "Name": "qMRLab",
+      "Version": "2.4.1",
+      "Container": {
+        "Type": "docker",
+        "Tag": "qmrlab/minimal:2.4.1"
+        }
+    },
+    {
+      "Name": "Manual",
+      "Description": "Generated example T1map outputs"
+    }
+  ],
+  "SourceDatasets": [
+    {
+      "DOI": "DOI 10.17605/OSF.IO/K4BS5",
+      "URL": "https://osf.io/k4bs5/",
+      "Version": "1"
+    }
+  ]
+}
+```
+
+In addition to the metadata fields provided in the `dataset_description.json`, qMRI maps
+are RECOMMENDED to be accompanied by sidecar JSON files that contain further information
+about the quantified maps. Although this may not be the generic case for common derivative
+outputs, a proper interpretation of qMRI maps may critically depend on some metadata
+fields. For example, without the information of `MagneticFieldStrength`, white-matter T1
+values in a `T1map` become elusive.
+
+* All the acquisition parameters that are constant across the files in a file collection
+are RECOMMENDED to be added to the sidecar json of the qMRI maps.
+* Relevant acquisition parameters that vary across files in a qMRI file collection are
+RECOMMENDED to be added to the sidecar json of the qMRI map **`in array form`**.
+* The JSON file accompanying a qMRI map which is obtained by using open-source software
+is RECOMMENDED to include additional metadata fields listed in the following table:
+
+| **Field name**                  | **Definition**                                                                                                               |
+| ------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `BasedOn`                       | List of files in a file collection to generate the map. Fieldmaps are also listed, if involved in the processing.            |
+| `EstimationReference`           | Reference to the study/studies on which the implementation is based.                                                         |
+| `EstimationAlgorithm`           | Type of algoritm used to perform fitting (e.g. linear, non-linear, LM etc.)                                                  |
+| `Units`                         | Units of the maps, in accordance with the BIDS specification.                                                                |
+
+Example:
+
+```Text
+sub-01_T1map.nii.gz
+sub-01_T1map.json
+```
+
+```
+sub-01_T1map.json
+{
+
+<<Parameter injected by the software/pipeline>>
+
+"BasedOn":["anat/sub-01_fa-1_VFA.nii.gz",
+           "anat/sub-01_fa-2_VFA.nii.gz",
+           "anat/sub-01_fa-3_VFA.nii.gz",
+           "anat/sub-01_fa-4_VFA.nii.gz",
+           "fmap/sub-01_TB1map.nii.gz"],
+"EstimationPaper":"Deoni et. al.MRM, 2015",
+"EstimationAlgorithm":"Linear",
+"Units": "second",
+
+<<Parameters that are constant across files in the (parent) file collection>>
+
+“MagneticFieldStrength”: “3”,
+“Manufacturer”: “Siemens”,
```

Comment:
pedantic comment incoming: please use `"` instead of `“`

Reply:
https://github.com/bids-standard/bids-specification/pull/690/commits/d16bd402834819f489317bb104383f76daac8358